luisa_cortezano/ejercicio.md

```diff
@@ -0,0 +1,34 @@
+```
+Sub recolecta()
```

Comment:
Please check your code, variables aren't initialized like this. You must initialize variables like this: tr = 0, sb = 0, and so on.